### ISSUE TITLE
Seems to hang forever if given no parameters

### ISSUE THREAD

**Issue by nico-abram** [mookid/diffr#5]
At least on window 7 `diffr` seems to hang forever.

**Comment by mookid** [comment]
This is intended. You are supposed to give it some text from stdin.

See the [README](https://github.com/mookid/diffr#how-to-use-it) or the command line help: `diffr -h` says 

```
USAGE:
    diffr reads from standard input and write to standard output.

    Typical usage is for interactive use of diff:
    diff -u <file1> <file2> | diffr
    git show | diffr
```

**Comment by nico-abram** [comment]
What about adding a flag to read from stdin? That way you can show the help msg if someone runs `diffr`

**Comment by carenas** [comment]
alternatively, PR #7 could help with this

**Comment by mookid** [comment]
Hi @nico-abram,

This is fixed by 83b1201.

@carenas thanks for the contribution!
